app/modules/asset_groups/schemas.py

```diff
@@ -27,11 +27,13 @@ class DetailedAssetGroupSightingSchema(BaseAssetGroupSightingSchema):
     Detailed Asset_group_sighting schema exposes all useful fields.
     """
 
+    assets = base_fields.Function(AssetGroupSighting.get_assets)
```

Comment:
Why did you decide to change this?

Reply:
I like functions to say what they are doing doing this allows the function to be called "get" assets rather than just assets without having the name of the schema entry also called get_assets